src/components/utils.css

```diff
@@ -9,8 +9,65 @@
 }
 .component-container {
   padding: 1rem;
-  margin: auto;
+  margin: 1rem auto;
   max-width: 80%;
   border: 1px solid var(--secondary-text);
   border-radius: 0.4rem;
+  display: flex;
+  flex-wrap:wrap;
+  gap: 1rem;
+}
+header {
+  /* display:flex; */
+  grid-area: header;
+  background: var( --primary-color);
+  color:white;
+  padding: 1rem;
+}
+aside {
+  grid-area: aside;
+  display:flex;
+  flex-direction: column;
+  align-items:center;
+  justify-content:center;
+}
+aside a{
+  padding: 1rem;
+  color:var(--secondary-color);
+  font-weight:500;
+}
+main {
+  grid-area: main;
+  overflow-y: auto;
+}
+body {
+  height: 100vh;
+  max-width: 100%;
+
+  display:grid;
+  grid-template-columns: 30rem 1fr 1fr ;
+  grid-template-rows: auto 1fr 1fr;
+  grid-template-areas:
+  "header header header"
+  "aside main main"
+  "aside main main";
+  gap:1rem;
+}
+a{
+  text-decoration:none;
+}
+ul{
+  list-style: none;
+  padding:0 1rem;
+}
+.list{
+  display:flex;
+  flex-direction: column;
+  margin:auto;
+  gap:1rem;
+}
+iframe body {
+  overflow-x: scroll important!;
+  width: fit-content important!;
```

Comment:
I think this would be !important. And if you could please avoid this attribute

Reply:
removed it